src/ServiceProvider.php

```diff
@@ -149,6 +149,9 @@ protected function registerPermissions(): self
                                     Permission::make("create {$resource->handle()}")
                                         ->label($this->permissionLabel('create', $resource)),
 
+                                    Permission::make("publish {$resource->handle()}")
```

Comment:
The concept of "publish states" are opt-in, so we probably only want to add this permission if they're enabled for the resource.

```php
if ($resource->hasPublishStates()) {
    Permission::make("publish {$resource->handle()}")
        ->label("Manage {$resource->handle()} Publish State"),
}
```